### DISCUSSION TITLE
Performance issue

### DISCUSSION THREAD

**Discussion by LouisH61** [mpv-player/mpv#15404] [Q&A]
I played a particular video and it lagged (low, inconsistent fps). It doesn't happen to other videos, and that particular video is fine when played using other players. Any troubleshooting tips?

**Comment by llyyr** [comment]
`--profile=fast`

**Reply by kasper93** [reply]
You should try `--hwdec` first. Profile fast does not include hardware decoding.

**Reply by LouisH61** [reply]
You are able to toggle hwdec with right right click menu right? I think I tried it but didn’t work.

**Reply by LouisH61** [reply]
The video I played is a 1080p movie, I also played another 4K movie and it turned out fine. Strange.
